gordon-plugin/src/main/kotlin/com/banno/gordon/GordonTestTask.kt

```diff
@@ -67,6 +69,9 @@ internal abstract class GordonTestTask @Inject constructor(
     @get:Input
     internal val tabletShortestWidthDp: Property<Int> = objects.property()
 
+    @get:Input
+    internal val ignoreProblematicDevices: Property<Boolean> = objects.property()
```

Comment:
I think this should be internal. When any property marked as an input changes, the task has to rerun. Think of this case: If, after a successful test run, the only change made is to the `ignoreProblematicDevices` property, I don't think the task should rerun because it doesn't affect the tests themselves.

```suggestion
    @get:Internal
    internal val ignoreProblematicDevices: Property<Boolean> = objects.property()
```